src/lualib/ArrayFind.ts

```diff
@@ -0,0 +1,14 @@
+// https://www.ecma-international.org/ecma-262/10.0/index.html#sec-array.prototype.find
+function __TS__ArrayFind<T>(this: void, arr: T[], callbackFn: (element: T, index?: number, array?: T[]) => boolean): T {
```

Comment:
```suggestion
function __TS__ArrayFind<T>(this: void, arr: T[], callbackFn: (element: T, index: number, array: T[]) => unknown): T {
```

Reply:
and the return type should probably be `T | undefined`